.github/actions/e2e-atomic-angular/action.yml

```diff
@@ -21,7 +21,7 @@ runs:
         wait-on-timeout: 600000
         install: false
         record: false
-    - uses: actions/upload-artifact@ff15f0306b3f739f7b6fd43fb5d26cd321bd4de5 # v3
+    - uses: actions/upload-artifact@50769540e7f4bd5e21e526ee35c689e35e0d6874 # v4
```

Comment:
I used the same hash as the upload-artifact v4 we already had in the repo

https://github.com/coveo/ui-kit/blob/master/.github/actions/merge-playwright-reports/action.yml